doc/ale.txt

```diff
@@ -3126,6 +3126,9 @@ documented in additional help files.
     ols...................................|ale-reasonml-ols|
     reason-language-server................|ale-reasonml-language-server|
     refmt.................................|ale-reasonml-refmt|
+  rego....................................|ale-rego-options|
```

Comment:
Since you added cpell support would be nice to have cpell added here. Same for the other index documents.

Reply:
@hsanson thanks for reviewing this PR, good catch, I've added the required docs now.